Resources/ServerInfo/_NF/Guidebook/Shipyard/Ambition.xml

```diff
@@ -19,24 +19,7 @@
 	[color=#a4885c]Available Roles:[/color] Pilot, Chief Engineer, Atmospheric Technician, Station Engineer, Salvage Specialist
 	
   "Declassified gas production and distribution platform seized in a hostile takeover of an Atmosian conglomerate. For the ultimate insurgent."
-  
-  # Piloting
-  <Box>
-    <GuideEntityEmbed Entity="ComputerShuttle"/>
-  </Box>
-  After clicking the shuttle console, you should see a radar view of the shuttle. Here are the steps for piloting the shuttle back and forth:
-
-  - First, disconnect any airlocks that are connected to the dock.
-  - Then, you actually get to pilot the shuttle. The controls are fairly simple, with [color=#028ed9]"W"[/color] and [color=#028ed9]"S"[/color] being forward and backward, [color=#028ed9]"A"[/color] and [color=#028ed9]"D"[/color] being left and right, and [color=#028ed9]"Q"[/color] and [color=#028ed9]"E"[/color] being rotating left and right; and [color=#028ed9]"spacebar"[/color] being the brake and moving precicely by holding the [color=#028ed9]"spacebar"[/color] while doing other inputs.
-  
-  # Hiring crew
-  <Box>
-    <GuideEntityEmbed Entity="ComputerStationRecords"/>
-    <GuideEntityEmbed Entity="ClothingHeadsetGrey"/>
-    <GuideEntityEmbed Entity="IntercomCommon"/>
-  </Box>
-  As a Captain of a Nanotrasen vessel, you have the authority to hire, fire, demote, or promote crew members at will. There are two ways you as a Captain can go about hiring crew: you can either use the [color=#a4885c]station records computer[/color] on your ship to open crew slots (jobs available vary from ship to ship) or invite through in-game communications other players to your crew.
-  
+    
   # PREFLIGHT CHECKLIST
```

Comment:
Just throwing this out as an idea.  The ships as-is spawn ready to fly (except the Construct with its unanchored generator).  Instead of "PREFLIGHT CHECKLIST", why not "TROUBLESHOOTING"?

Reply:
There will at some point be a dedicated troubleshooting guidebook entry, but for now something needs to exist.

Once the new entry is done we can look at going through and removing everything but the most ship-specific setup instructions.